acceptance/tests/resource/zone/ticket_4840_should_create_zone_with_zpool.rb

```diff
@@ -18,19 +18,24 @@
     zfs { "rpool/tstfs":
       mountpoint => "/ztstpool/mnt",
       ensure => present,
+      require => File["/ztstpool/mnt"],
+    }
+    file { "/ztstpool":
+      ensure => directory,
+      mode => "0755",
     }
     file { "/ztstpool/mnt":
       ensure => directory,
       mode => "0700",
-      require => Zfs["rpool/tstfs"],
+      require => File["/ztstpool"],
```

Comment:
The file will autorequire its parent, so requiring the parent isn't necessary. But bigger question is why are we no longer requiring `Zfs["rpool/tstfs"]`?